### PR TITLE
Mac/Linux, Tray Minimize, Active Sounds, & Better Error Handling

### PR DESCRIPTION
**Pull request by DeekyJay**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -103,15 +103,24 @@ app.on('ready', () => {
   })
 })
 
+const WIN32 = process.platform === 'win32'
+
 ipcMain.on('CHECK_FOR_UPDATE', function (event) {
   console.log('CHECKING')
```
Comment (alfw) [inline main.js:109]: Left in debug code?

File: main.js [hunk 2]
```diff
@@ -103,15 +103,24 @@ app.on('ready', () => {
...
-  updater.check((err, status) => {
-    console.log(err, status)
-    if (!err && status) {
-      // Download the update
-      updater.download()
-    }
-  })
+  if (WIN32) {
+    updater.check((err, status) => {
+      console.log(err, status)
```
Comment (alfw) [inline main.js:112]: Left in Debug Code?

File: main.js (hunk 1 above)
Comment (DeekyJay) [reply main.js:109]: Currently in the beta process, I leave console logs in as my error reporting system shows a breadcrumb trail of logs. This helps me understand how to reproduce certain errors.

File: main.js (hunk 2 above)
Comment (DeekyJay) [reply main.js:112]: Currently in the beta process, I leave console logs in as my error reporting system shows a breadcrumb trail of logs. This helps me understand how to reproduce certain errors.

**Pull request merged**